pkg/controllers/cache.go

```diff
@@ -81,7 +81,7 @@ func (n *objectWatcher) watch(ctx context.Context, obj client.Object) error {
 
 	// Add an event handler that only allows events through for the correct object name
 	// Since the informer is namespace bound, this should limit the events from this event handler to a single resource.
-	informer.AddEventHandler(&eventToChannelHandler{
+	_, _ = informer.AddEventHandler(&eventToChannelHandler{
```

Comment:
i suppose `AddEventHandler` returns an error, can we propagate this instead suppressing?

Reply:
This change will be covered in an upcoming PR for k8s packages update. This is not an issue with the old version k8s having deps with older version golangci-lint.